CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ See updating [Changelog example here](https://keepachangelog.com/en/1.0.0/)
 
 - upcloud_loadbalancer_frontend: handle changes in the `networks`
 - upcloud_loadbalancer: set `maintenance_dow` and `maintenance_time` as computed to avoid planning them to be removed when missing from configuration.
+- upcloud_storage: sync title length constraint with API, allows 1-255 characters now
```

Comment:
This is now under _5.8.0_ instead of _Unreleased_ 🕵️‍♂️

Reply:
Damn, I did not have a time machine after all. Force-fixed in 4b90c68c65588a439d9712aa680fe0b87e567e5b